core/core.go

```diff
@@ -441,6 +443,23 @@ func (t *TextileNode) WaitForRoom() {
 	}
 }
 
+func (t *TextileNode) GatewayPort() (int64, error) {
```

Comment:
sweet... just `int64` -> `int`, no?

Reply:
yeh, i think `strconv.ParseInt` requires a bit size, which we have as 64. So maybe this creates an int64? I think its easy enough to cast on return, so i'll add that

Reply:
done